PocketKit/Sources/PocketKit/Recommendation/Recommendation+Extenstions.swift

```diff
@@ -3,6 +3,8 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import Foundation
+import UIKit
```

Comment:
Is UIKit required to call the CDNURLBuilder?

Reply:
probably an automatic import, in any case those two together are redundant, will fix.